core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -437,6 +492,37 @@ class AdminManager(val config: KafkaConfig,
     }.toMap
   }
 
+  private def prepareConfigs(alterConfigOps: List[AlterConfigOp], configProps: Properties, configKeys: Map[String, ConfigKey]): Unit = {
```

Comment:
nit: What do you think about a more descriptive name like `prepareIncrementalConfigs`?